newIDE/app/src/Profile/CreateAccountDialog.js

```diff
@@ -102,19 +114,23 @@ const CreateAccountDialog = ({
     !isValidatingUsername &&
     (!usernameAvailability || usernameAvailability.isAvailable);
 
-  const createAccount = () => {
+  const createAccount = async () => {
     if (!canCreateAccount) return;
-    onCreateAccount({
-      email,
-      password,
-      username,
-      getNewsletterEmail,
-    });
+    try {
+      await onCreateAccount({
+        email,
+        password,
+        username,
+        getNewsletterEmail,
+      });
+    } catch (error) {
+      console.error(error);
+    }
   };
 
   return (
     <Dialog
-      title={<Trans>Create a new GDevelop account</Trans>}
+      title={null}
```

Comment:
Since you added a comment for each dialog that does not have a title, should we add one: `This dialog has a custom design to be more welcoming, the title is set in the content`